src/accel/cuda/clone.cu

```diff
@@ -98,11 +98,12 @@ void clone_track(Track* track_h, dev_track* track_d,
   new_track._uid = track_h->getUid();
   new_track._num_segments = track_h->getNumSegments();
   new_track._azim_angle_index = track_h->getAzimAngleIndex();
-  new_track._refl_in = track_h->isReflIn();
-  new_track._refl_out = track_h->isReflOut();
-  new_track._bc_in = track_h->getBCIn();
-  new_track._bc_out = track_h->getBCOut();
+  new_track._next_in = track_h->isNextIn();
+  new_track._next_out = track_h->isNextOut();
 
+  new_track._bc_in = std::min((int)track_h->getBCIn(), 1);
```

Comment:
Sounds like good reasoning to me.
On Aug 15, 2015 4:20 PM, "sam" <notifications@github.com> wrote:

> In src/accel/cuda/clone.cu
> <https://github.com/mit-crpg/OpenMOC/pull/185#discussion_r37140472>:
>
> >
> > +  new_track._bc_in = std::min((int)track_h->getBCIn(), 1);
>
> How about just an if/else? If the bc is vacuum, set bc to 0; else set bc
> to 1. This will work for the current BC options and will work for
> rotational BCs, if we ever decide to implement them. The only other BC I
> can think of is white BCs and I'm not sure how they would be implemented,
> but I don't see any need to even consider those BCs at the moment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/OpenMOC/pull/185/files#r37140472>.
>